guide/src/SUMMARY.md

```diff
@@ -26,6 +26,7 @@
   - [web-sys: WebAudio](./examples/web-audio.md)
   - [web-sys: WebGL](./examples/webgl.md)
   - [web-sys: A Simple Paint Program](./examples/paint.md)
+  - [Parallel Raytracing](./examples/raytrace.md)
```

Comment:
Well, synchronisation is involved, it's more concurrent than parallel, but I don't really mind.

Reply:
This is nittiest of nitpicking but:

As long as there are multiple workers that are running on different cores, then this *is* parallelism.

Just because there are synchronization points doesn't make something not parallel.

concurrent = operations in-progress (but maybe not doing work) at the same time

parallel = doing work at the same time

For example, threading is always concurrent, but it might not be parallel if there is only a single core that is only actually working on one thread at any given moment and is time slicing between all of the threads.